### PR TITLE
update to JSON Forms 3.0.0

### PR DESCRIPTION
**Pull request by LukasBoll**
Signed-off-by: Lukas Boll lukas-bool@web.de

### CODE REVIEW

File: src/components/common/Demo.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import Highlight, { defaultProps } from "prism-react-renderer";
 import { usePrismTheme } from '@docusaurus/theme-common';
+import get from 'lodash/get';
```
Comment (lucas-koehler) [inline src/components/common/Demo.js]: As I see it, this is not used in this file. Can we remove it?

File: src/components/examples/Categorization.js [hunk 2]
```diff
@@ -1,9 +1,22 @@
 import React from 'react';
 import { categorization, stepper, steppershownav } from '@jsonforms/examples';
 import { Demo } from '../common/Demo';
+import get from 'lodash/get';
+
+const translations = {
+  categoryLabelKey: 'Basic',
+  address: {
+    label: "Address",
```
Comment (lucas-koehler) [inline src/components/examples/Categorization.js]:
```suggestion
    label: 'Address',
```

**Review by lucas-koehler — CHANGES_REQUESTED** [review]
The styling seems to be broken for the new api. See the following screenshot showing the new core api on the left and the current website on the right:
image

It is suspicious that the PR deletes many `main.css` files, e.g. [static/api/core/assets/css/main.css](https://github.com/eclipsesource/jsonforms2-website/pull/240/files#diff-8a6f0337e722ffdf6a265a79dd5c0edce56ce8b2ed27379457ca91c2320c475e)

Maybe something went wrong during api generation or copying?

**Comment by LukasBoll** [comment]
Thanks for the review. I think the css classes are not deleted, but the css files are collapsed into one line. The styling is working on the [deploy preview](https://deploy-preview-240--jsonforms.netlify.app/api/core/index.html). I checked the styling local again: When running `npm run start` everything worked for me, while running `npm run serve` was leading to the results you mentioned.

**Review by lucas-koehler — CHANGES_REQUESTED** [review]
Hi @LukasBoll , thanks for the update and the explanation. The behavior is fine with me as you explained. Maybe the serve just behaves strangely.
However, I noticed one more issue with the new page: The title of the new API pages is only `'JSON'` now. See this screenshot of the core API (the problem is also present for other APIs)
image

**Review by lucas-koehler — APPROVED** [review]
Approved because fixing the API page titles will be fixed in follow up issue #241

**Pull request merged**